src/components/allComments.js

```diff
@@ -9,6 +9,8 @@ import {
 } from '@material-ui/core'
 import { SmallRating } from '../app/mycomments'
 import Img from 'gatsby-image'
+import { navigateTo } from 'gatsby'
```

Comment:
That is deprecated, should use `navigate` instead

Reply:
okay sure